zebra-chain/src/transparent/tests/vectors.rs

```diff
@@ -0,0 +1,19 @@
+use super::super::serialize::parse_coinbase_height;
+
+#[test]
+fn parse_coinbase_height_mins() {
```

Comment:
Test Coverage: Can you please the height `1` examples from the ticket as well?

We need to cover heights `1..=16`, which have a one-byte encoding.

Reply:
Sure, added them at https://github.com/ZcashFoundation/zebra/pull/3129/commits/73907af994d587b420c653b659fa58d4113cb9d0